app/src/main/java/com/example/fooding/adapters/SelectedIngredientsAdapter.java

```diff
@@ -37,15 +40,18 @@ public int getItemCount() {
     }
 
     public class ViewHolder extends RecyclerView.ViewHolder {
-        TextView ingredientNameTextView;
+        TextView tvIingredientName;
+
 
         public ViewHolder(@NonNull View itemView) {
             super(itemView);
-            ingredientNameTextView = itemView.findViewById(R.id.ingredientName);
+            tvIingredientName = itemView.findViewById(R.id.ingredientName);
```

Comment:
Maybe because i am out of context, but what does tVIingredientName refer to? I also see IVIingredientName?

Reply:
That was the id I gave to the ingredient name and image, but I modified the naming already. i will commit and push the updated code